realm/realm-jni/src/io_realm_internal_TableView.cpp

```diff
@@ -148,10 +150,7 @@ JNIEXPORT void JNICALL Java_io_realm_internal_TableView_nativePivot(
 JNIEXPORT void JNICALL Java_io_realm_internal_TableView_nativeClose(
     JNIEnv*, jclass, jlong nativeViewPtr)
 {
-    if (nativeViewPtr == 0)
```

Comment:
This code is not transferred to the new finalize_table_view(). Are you ensuring nativeViewPtr is not 0 somewhere else?